README.md

```diff
@@ -6,10 +6,57 @@ This protocol is built to decouple the backend and frontend of trace analysers,
 
 The protocol is meant to be RESTful, over HTTP.
 
-The specification is currently written in **OpenAPI 3.0** and can be pretty-visualized in the [github pages](https://theia-ide.github.io/trace-server-protocol/).
+The specification is currently written in **OpenAPI 3.0** and can be pretty-visualized in the [github pages][6].
 
 ## HOW-TO
 
-The specification should be edited with the [OpenAPI (Swagger) Editor extension](https://marketplace.visualstudio.com/items?itemName=42Crunch.vscode-openapi) for VS Code.
+The specification should be edited with the [OpenAPI (Swagger) Editor extension][8] for VS Code.
 
 The latter extension is assumed for consistent formatting of the `./API.yaml` file over time.
+
+### Setup
+
+To initialize a local virtual environment, type the following commands in the root directory:
+
+```shell
+python3 -m venv .venv
+source .venv/bin/activate
+pip install -r requirements.txt
+```
+
+The virtual environment can be replaced with another local setup.
+
+### Alternate TSP version
+
+Above, `API.yaml` has been the manually documented TSP version up until now.
+
+* That version is the one shown by the default [github pages][6].
+* That version does not fully match what trace-server currently supports; some endpoints differ or are missing in server.
+* Aligning the TSP documented that way with server's own endpoints is a work in progress.
+
+In the meantime, Swagger has recently been added to trace-server.
```

Comment:
The Trace Compass trace server is an example implementation or reference implementation of the TSP server side. In general the TSP is independent to any specific client or server implementation. Could please clarify that and that you referring to the "Trace Compass trace server" here as a reference implementation which is used to refine the TSP?